tests/test_api.py

```diff
@@ -1,21 +1,22 @@
+import collections
 import os
-import pyotp
 import shutil
 import tempfile
 import time
 import unittest
+
+import pyotp
 import vcr
+from utils import load_auth
+from utils import save_auth
 
 from sdclientapi import API
-from sdclientapi.sdlocalobjects import (
-    BaseError,
-    WrongUUIDError,
-    ReplyError,
-    Reply,
-    Source,
-)
-from utils import load_auth, save_auth
-
+from sdclientapi.sdlocalobjects import AuthError
+from sdclientapi.sdlocalobjects import BaseError
+from sdclientapi.sdlocalobjects import Reply
+from sdclientapi.sdlocalobjects import ReplyError
+from sdclientapi.sdlocalobjects import Source
+from sdclientapi.sdlocalobjects import WrongUUIDError
```

Comment:
These changes are actually not required, the style of import you see is actually from `black` modifying the source import lines.

Reply:
From isort, actually, but black would do The Right Thing(TM) too. :wink: